src/main/java/com/powsybl/openloadflow/network/impl/ContingencyTripping.java

```diff
@@ -59,6 +59,23 @@ public static ContingencyTripping createBranchTripping(Network network, Branch<?
         }
     }
 
+    public static ContingencyTripping createTieLineTripping(Network network, TieLine line, String voltageLevelId) {
```

Comment:
It is always called with null as voltage level?

Reply:
yes, is it an issue?